src/apprt/gtk/App.zig

```diff
@@ -972,6 +972,9 @@ fn loadRuntimeCss(
     const headerbar_foreground = config.@"window-titlebar-foreground" orelse config.foreground;
 
     try writer.print(
+        \\window.without-window-decoration-and-with-titlebar {{
+        \\  border-radius: 0 0;
+        \\}}
         \\widget.unfocused-split {{
```

Comment:
I think that this would be better in the static `style.css` since it never changes.

Reply:
Probably right. I'd accept a follow up on it, but also fine with it here.

Reply:
Whoops my bad. I forgot about that. I will do that.